src/index.html

```diff
@@ -0,0 +1,16 @@
+<!doctype html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport"
+        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>Colored Blocks</title>
+  <link rel="stylesheet" href="style.css">
+</head>
+<body>
+  <div class="first"></div>
```

Comment:
In this task ok, but for the future, `first, second, third ...` bad name for the classes.